clinica/pydra/t1_volume/utils.py

```diff
@@ -0,0 +1,72 @@
+from os import PathLike
+from typing import Tuple, Union
+
+
+def initialize_tissues_spm_segment(
+    tissue_probability_map: Union[PathLike, None] = None
+) -> tuple:
+
+    """Prepare data structure for SPM segment interface
+
+    Parameters
+    ----------
+    tissue_probability_map: PathLike, optional
+        Path to the nifti tissue probability map. Default=None.
+
+    Returns
+    -------
+    tuple
+        "tissues" data structure for SPMSegment
+    """
+
+    from clinica.pipelines.t1_volume_tissue_segmentation.t1_volume_tissue_segmentation_utils import (
+        get_tissue_tuples,
+    )
+    from clinica.utils.spm import get_tpm
+
+    parameters = {}
+    parameters.setdefault("tissue_classes", [1, 2, 3])
+    parameters.setdefault("dartel_tissues", [1, 2, 3])
+    parameters.setdefault("save_warped_unmodulated", True)
+    parameters.setdefault("save_warped_modulated", False)
+
+    if tissue_probability_map:
+        parameters.setdefault("tissue_probability_maps", tissue_probability_map)
+    else:
+        parameters.setdefault("tissue_probability_maps", get_tpm())
+
+    tissue_tuples = get_tissue_tuples(
+        parameters["tissue_probability_maps"],
```

Comment:
I don't understand why you need to go through the parameters dictionary here? Why not just return the tuple with the values you set as defaults?